README.md

```diff
@@ -1,91 +1,106 @@
-# FCL Dev Wallet
+# FCL Dev-Wallet
 
-**IMPORTANT**
+A Flow wallet for development purposes. To be used with the Flow Emulator.
 
-```
-The FCL Dev Wallet is currently in active development and
-it's current state should be considered as a preview.
+## Introduction
 
-If you encounter any problems please make sure you are
-using an up-to-date copy and if the issue persists please
-let us know by creating an issue.
+The FCL `dev-wallet` is an `fcl` [(Flow client-library)](https://github.com/onflow-flow-js-sdk) compatible Flow wallet that simulates the protocols used by `@onflow/fcl` to interact with the Flow blockchain on behalf of Flow accounts.
```

Comment:
I'm not sure I like the hyphenated stylization of the dev wallet. I'd prefer to refer to tools by their full name "i.e. FCL dev wallet", "FCL" rather than in code brackets.

Reply:
```suggestion
The FCL dev wallet is a mock Flow wallet that simulates the protocols used by [FCL](https://docs.onflow.org/fcl/) to interact with the Flow blockchain on behalf of simulated user accounts.
```